pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -113,6 +113,10 @@ func convertToIstioAuthnFilterConfig(jwtRules []*v1beta1.JWT) *authn_filter.Filt
 // ensure Authn Filter won't reject the request, but still transform the attributes, e.g. request.auth.principal.
 // proxyType does not matter here, exists only for legacy reason.
 func (a *v1beta1PolicyApplier) AuthNFilter(proxyType model.NodeType, isXDSMarshalingToAnyEnabled bool) *http_conn.HttpFilter {
+	if len(a.processedJwtRules) == 0 {
+		log.Infof("RequestAuthentication (beta policy) not found, fallback to alpha if available")
```

Comment:
Infof -> Debugf

Reply:
done, added prefix to be able to distinguish line as well.